### PR TITLE
ASoC: SOF: mediatek: mt8195 change ipc timeout as 1 second

### PR DESCRIPTION
**Pull request by yaochunhung**
Hi,
This PR is used to resolve ipc timeout for mt8195 during platform booting.
DSP ipc transmission is over than default 500ms during platform boot
flow then trigger exception. Check the time between DSP send ipc complete
notification via mailbox interrupt and linux receive and handle the irq
in the function "mt8195_dsp_handle_reply". Generally it takes more than
500ms but less than one second. Therefore adjust the ipc_timeout as 1
second to avoid the heavy loading of system to cause ipc timeout.
Please help to review it. Thanks.

### CODE REVIEW

**Review by cujomalainey — APPROVED** [review]

**Review by ranj063 — DISMISSED** [review]

**Review by ujfalusi — CHANGES_REQUESTED** [review]
@yaochunhung, the commit message is really hard to follow.
It can be as simple as something like:
The IPC message sending can take longer than the default 500ms during system boot up due to the concurrent loading of different drivers.
Increase the IPC timeout to 1 second to avoid timeout errors due to Linux load and scheduling.

While at it, I would also instrument the system to see what blocks the boot for this long, there might be some misbehaving driver which can be fixed to have snappier bootup.

**Comment by yaochunhung** [comment]
@ujfalusi Thanks for your suggestion. I have updated the commit message. please help to review it. thanks.

**Review by cujomalainey — APPROVED** [review]

**Review by ranj063 — APPROVED** [review]

**Comment by yaochunhung** [comment]
SOFCI TEST

**Review by ujfalusi — APPROVED** [review]

**Review by plbossart — APPROVED** [review]
